package.json

```
@@ -36,6 +36,7 @@
     "build-frontend-docs": "rimraf ./docs/frontend_api && jsdoc -c jsdoc-conf.json -d ./docs/frontend_api src/public/app/entities/*.js src/public/app/services/frontend_script_api.js src/public/app/widgets/basic_widget.js src/public/app/widgets/note_context_aware_widget.js src/public/app/widgets/right_panel_widget.js",
     "build-docs": "npm run build-backend-docs && npm run build-frontend-docs",
     "webpack": "cross-env node --import ./loader-register.js node_modules/webpack/bin/webpack.js -c webpack.config.ts",
+    "test-playwright": "npx playwright test",
```

I don't think we need to explicitly call `npx` in `package.json` scripts.

```suggestion
    "test-playwright": "playwright test",
```

---

you're right :-) I've ammended my commit accordingly